src/.gitignore

```diff
@@ -5,3 +5,6 @@
 *.o
 *.a
 *.lib
+
+## VS Code
+../.vscode/
```

Comment:
I didn't realize I put the `.gitignore` in a sub-directory. We might as well move that up to the root.